tests/gocase/unit/type/zset/zset_test.go

```diff
@@ -1308,6 +1324,10 @@ func basicTests(t *testing.T, rdb *redis.Client, ctx context.Context, enabledRES
 		}
 		createZset(rdb, ctx, "zset", z)
 
+		// ZRANDMEMBER zset
+		str := rdb.Do(ctx, "ZRANDMEMBER", "zset").Val()
```

Comment:
Can use `rdb.ZRandMember`.

Reply:
Seems not a major issue, maybe we can merge it first.

Reply:
Sorry, it can't use rdb.ZRandMember, because this function have a `count` param,
ZRandMember(ctx context.Context, key string, count int);
and  this bug must not input `count` param;

Reply:
![image](https://github.com/user-attachments/assets/e5f118da-df2e-45ab-93d2-c6f8aa6b222e)

Reply:
I see, thank you!